### ISSUE TITLE
Review security headers again

### ISSUE THREAD

**Issue by apertome** [perfsonar/graphs#104]
We got a report from a user on the user list that their graphs weren't loading. Use case was: MA and Maddash separate host. They had their maddash embedding graphs on the maddash host, but I suggested they embed the graphs from the MA host instead. 

Currently, /perfsonar-graphs/ has to be on the same host as the MA, which we should give more consideration.

**Comment by arlake228** [comment]
Just a datapoint. On http://ps-dashboard.es.net the MaDDash server is separate from all the MAs. We run an MA on each of our measurement hosts so there is close to 50 remote ones MaDDash is talking to. After upgrading to 4.0.1, the speed that the graphs load was pretty slow..almost unusable really. Turns out due to the security policy headers, it had to fallback to graphData.cgi which seemed to be a pretty slow process. After I removed the two instances of `Header set Content-Security-Policy...` from `/etc/httpd/conf.d/apache-perfsonar-graphs.conf` things got much better. Next time perfsonar-graphs updates this change will get overwritten. I think this is a very common use case, in fact I'd expect it to be more common than MaDDash and the MA on the same host, so we should really explore whether these headers makes sense.
